aptos-move/framework/move-stdlib/sources/vector.move

```diff
@@ -24,6 +26,11 @@ module std::vector {
     /// The range in `slice` is invalid.
     const EINVALID_SLICE_RANGE: u64 = 0x20004;
 
+    /// Whether to utilize native vector::move_range
+    /// Vector module cannot call features module, due to cyclic dependency,
+    /// so this is a constant.
+    const USE_MOVE_RANGE: bool = true;
```

Comment:
native functions flag hasn't yet been enabled in mainnet. If it isn't enabled before this framework goes out - I need to disable it. Easier to create a diff that just sets here to false, instead of changing all the functions. 

once flag is enabled, will clean up the code.